array.c

```diff
@@ -935,21 +935,26 @@ rb_check_to_array(VALUE ary)
 
 /*
  *  call-seq:
- *     Array.try_convert(obj) -> array or nil
+ *    Array.try_convert(obj) -> new_array or nil
  *
- *  Tries to convert +obj+ into an array, using the +to_ary+ method.  Returns
- *  the converted array or +nil+ if +obj+ cannot be converted.
- *  This method can be used to check if an argument is an array.
+ *  Tries to convert <tt>obj</tt> to an \Array.
  *
- *     Array.try_convert([1])   #=> [1]
- *     Array.try_convert("1")   #=> nil
+ *  When <tt>obj</tt> is an
+ *  {Array-convertible object}[doc/implicit_conversion_rdoc.html#label-Array-Convertible+Objects],
```

Comment:
What do you think about adding "(implements #to_ary)" to this line to summarize what's behind this link?

Reply:
Done, throughout.